uavcan/driver/python_can.py

```diff
@@ -34,20 +34,15 @@
 class PythonCan(AbstractDriver):
     TX_QUEUE_SIZE = 1000
 
-    def __init__(self, interface, **_extras):
+    def __init__(self):
         super(PythonCan, self).__init__()
 
         if can is None:
             logger.error("To use this driver, make sure the module python-can is installed")
             raise ValueError("python-can must be installed.")
 
-        # get bus from default config
         try:
-            if interface is None:
-                self._bus = can.Bus()
-            else:
-                self._bus = can.interface.Bus(channel=interface, bustype=_extras['bustype'], bitrate=_extras['bitrate'])

```

Comment:
@windelbouwman Why delete this?

Reply:
The interface in this case will be `pythoncan`, which is an invalid channel. We choose `pythoncan` as interface, so it makes no sense to pass it. For the second and third argument, I was unable to make it work, and I also do not use it yet. In any case, it is not required to check if interface is `None`.

Reply:
Thanks for your reply; `interface` was a variable that referred to the underlying CAN interface, e.g., can0 or vcan0, not to `pythoncan`. Regarding the other parameters, they are needed as well when not using an INI file.

> I would like to merge this driver first as it is, we can later refine it?

OK, no problem, I'll submit a PR with these changes afterwards.